### PR TITLE
(heal preprod) STRIDES + new hatchery sidecar

### PR DESCRIPTION
**Pull request by jawadqur**

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
# preprod.healdata.org/manifest.json
## :warning: Services on branch
- hatchery
- portal
## :warning: Services are being downgraded
- portal

**Comment by PlanXCyborg** [comment]
# preprod.healdata.org/manifest.json
## :warning: Services on branch
- hatchery
- portal
## :warning: Services are being downgraded
- portal

**Comment by PlanXCyborg** [comment]
# preprod.healdata.org/manifest.json
## :warning: Services on branch
- hatchery
- portal
## :warning: Services are being downgraded
- portal

**Comment by mfshao** [comment]
we also need to set `stridesPortalURL` in portal config? although we didn;t do that for qa-heal, we probebly forgot to do so

File: preprod.healdata.org/manifest.json [hunk 1]
```diff
@@ -14,13 +14,13 @@
...
     "fluentd": "fluent/fluentd-kubernetes-daemonset:v1.2-debian-cloudwatch",
-    "hatchery": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/hatchery:2022.04",
+    "hatchery": "quay.io/cdis/hatchery:feat_multiple_paymodels",
     "indexd": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/indexd:2022.04",
     "manifestservice": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/manifestservice:2022.04",
     "metadata": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/metadata-service:2022.04",
     "peregrine": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/peregrine:2022.04",
     "pidgin": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/pidgin:2022.04",
-    "portal": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/data-portal:3.22.1",
+    "portal": "quay.io/cdis/data-portal:feat_multiple_paymodels",
```
Comment (mfshao) [inline preprod.healdata.org/manifest.json:23]:
```suggestion
    "portal": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/data-portal:feat_multiple_paymodels",
```

**Comment by PlanXCyborg** [comment]
# preprod.healdata.org/manifest.json
## :warning: Services on branch
- hatchery
## :warning: Services are being downgraded
- portal

**Review by mfshao — APPROVED** [review]

**Comment by PlanXCyborg** [comment]
# preprod.healdata.org/manifest.json
## :warning: Services on branch
- hatchery
## :warning: Services are being downgraded
- portal

**Comment by mfshao** [comment]
already up to date, no longer needed

**Pull request closed without merging**